spec/docker_support.rb

```diff
@@ -0,0 +1,19 @@
+module DockerSupport
+  require 'cheffish/rspec/chef_run_support'
+  def self.extended(other)
+    other.extend Cheffish::RSpec::ChefRunSupport
+  end
+
+  require 'chef/provisioning/docker_driver'
+
+  def with_docker(description, *tags, &block)
+    docker_driver = Chef::Provisioning.driver_for_url("docker")
+
+    @@driver = docker_driver
+    def self.driver
+      @@driver
+    end
+
+    module_eval(&block)
+  end
```

Comment:
Tried cargo-culting `when_the_repository` from `AWSSupport`, but that didn't work either.